src/CMakeLists.txt

```diff
@@ -224,7 +222,7 @@ add_subdirectory(cppcache)
 add_subdirectory(cryptoimpl)
 add_subdirectory(dhimpl)
 add_subdirectory(sqliteimpl)
-add_subdirectory(gfcpp)
+add_subdirectory(getversion)
```

Comment:
Eeeek! This is the best name we could come up with for this stupid util? May we should just delete it?